pkg/cloud/scope/machinepool.go

```diff
@@ -43,11 +43,14 @@ import (
 	"sigs.k8s.io/cluster-api/util/patch"
 )
 
+const ReplicasManagedByAutoscalerAnnotation = "cluster.x-k8s.io/replicas-managed-by-autoscaler"
```

Comment:
Am I correct that we'd import this value from CAPI once [this PR](https://github.com/kubernetes-sigs/cluster-api/pull/7107) is merged rather than hardcoding it in here?

Reply:
yes, I think that would make sense.